src/main/java/generator/DecisionGenerator.java

```diff
@@ -36,13 +36,23 @@ public void generate(Model model){
         model.getScenario().addHas_decision(decision_3);
         actionByDecision.put(decision_3, action_3);
 
+        Decision decision_4 = factory.createDecision(ObjectNamer.getName("decision"));
+        Stop action_4 = factory.createStop(ObjectNamer.getName("stop"));
+        decision_4.addHas_action(action_4);
+        model.getScenario().addHas_decision(decision_4);
+        actionByDecision.put(decision_4, action_4);
```

Comment:
Wiem, że w samym środku różni się wywołaną metodą, ale to jest 4 prawie identyczny blok kodu, może da się to jakoś wyciągnąć do osobnej funkcji.

Reply:
Zrobione.